src/tokens/Token.jsx

```diff
@@ -518,10 +516,6 @@ const Token = ({
                        </text>
                       );
         }
-
-        let outlineColor = is(String, outline) ? c(outline) : 
```

Comment:
This worries me. It looks like you also removed the ability to see the outline color with this change?

Reply:
That was the code causing light gray circles to appear in the cities with squares even without a token being specified. The stations are on top so it was never shown before.